pkg/crc/machine/bundle/metadata.go

```diff
@@ -116,19 +118,44 @@ func (bundle *CrcBundleInfo) GetOpenshiftVersion() string {
 	return bundle.ClusterInfo.OpenShiftVersion
 }
 
-func (bundle *CrcBundleInfo) CheckDiskImageSize() error {
+func (bundle *CrcBundleInfo) Verify() error {
+	files := []string{
+		bundle.resolvePath(constants.OcExecutableName),
+		bundle.resolvePath(bundle.ClusterInfo.KubeadminPasswordFile),
+		bundle.GetKubeConfigPath(),
+		bundle.GetSSHKeyPath(),
+		bundle.GetDiskImagePath(),
+		bundle.GetKernelPath(),
+		bundle.GetInitramfsPath(),
+	}
+
+	for _, file := range files {
+		if file == "" {
+			continue
+		}
+		if _, err := os.Stat(file); err != nil {
+			if os.IsNotExist(err) {
+				return fmt.Errorf("%s not found in bundle", filepath.Base(file))
```

Comment:
This seems like a good candidate for a wrapped error?

Reply:
I wanted to put a more friendly error otherwise the user would see "stat /longpath/file: no such file or directory".

Reply:
Friendly error is very nice, but information was lost on the way ("file was missing"), which could be useful to users of this API. We probably should start adding a `UserFriendlyError { friendlyMessage string, userUnfriendlyError error }`, and only accept this error type in the upper layers. Just food for thoughts, let's keep things as is  for now.